mobile/cmp/navigator/Navigator.scss

```diff
@@ -1,4 +1,39 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+$size: 36px;
+$half-size: 18px;
+
 .xh-navigator {
-  position: relative;
-  flex: 1;
-}
+  &__swipe-indicator {
+    margin: -$half-size 0 0;
+    position: absolute;
+    top: 50%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: $size;
+    height: $size;
+    color: var(--xh-appbar-color);
+    background-color: var(--xh-appbar-bg);
+    border: 1px solid var(--xh-appbar-border-color);
+    border-radius: $half-size;
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
+    z-index: 21;
```

Comment:
How did you land on this z-index value?

Reply:
This is the z-index used throughout the mobile toolkit in popups, toasts, etc to mean "show above everything else, including masks". IIRC, the origin comes from showing above react-selects dropdowns